README.md

```diff
@@ -22,6 +22,7 @@ walk you through building your first Slack app using Node.js.
 | Web API      | Send data to or query data from Slack using any of [over 130 methods](https://api.slack.com/methods). | [`@slack/web-api`](https://slack.dev/node-slack-sdk/web-api) |
 | Events API   | Listen for incoming messages and [many other events](https://api.slack.com/events) happening in Slack, using a URL. | [`@slack/events-api`](https://slack.dev/node-slack-sdk/events-api) |
 | Interactive Messages | Respond to button clicks, dialogs, and other interactions with messages. | [`@slack/interactive-messages`](https://slack.dev/node-slack-sdk/interactive-messages) |
+| OAuth        | Setup the authentication flow using OAuth 2.0 for Slack apps and OAuth 1.0 for classic Slack apps. | [`@slack/oauth`](https://slack.dev/node-slack-sdk/oauth) |
```

Comment:
As these versions of OAuth are very Slack-specific, saying OAuth 1.0 and OAuth 2.0 may be confusing.

When you say OAuth 1.0, people remind [the OAuth 1.0 protocol](https://tools.ietf.org/html/rfc5849) (It's a bit nostalgic!). How about `OAuth 2.0 for Slack apps (both the latest and classic models)` or similar. I would like to hear from others for more natural expressions.

Reply:
@seratch yeah I can see how that is confusing. We might be better off using `V2 OAuth for Slack Apps as well as V1 OAuth for Classic Slack apps`. This is the wording in the readme for OAuth. 

v1 OAuth for classic slack apps is still OAuth 2.0.

Reply:
So, do you mean `Setup the authentication flow using V2 OAuth for Slack Apps as well as V1 OAuth for Classic Slack apps` in a full sentence? Looks great to me 👍

Reply:
Thanks for the discussion @seratch and @stevengill!

I've updated the description to be the full sentence suggested by @seratch - based on the wording from the `README.md` for OAuth. See commit 1b7e92a7bfee7cc253f240084cb2dd0e9d613f40

_From the perspective of someone new to the Slack API and documentation_, I found "V2 OAuth and V1 OAuth" to be misleading. I immediately assumed it was "OAuth 2.0 and OAuth 1.0" written in a non-standard way. Like @seratch mentioned, I thought it was a little surprising that Slack once used OAuth 1.0, haha!

It may be worth creating another issue to discuss updating the OAuth documentation to clarify that it's OAuth 2.0 for Slack apps (both latest and classic Slack apps). If you think this is a worthwhile task, I'd be happy to take a stab at it. ✋🏻

Reply:
Co-signing this -- I had the exact same reaction/interpretation when I read "V2 OAuth and V1 OAuth".

Reply:
@mwbrooks feel free to open an issue for clarifying the wording and take a stab at it! I've approved the PR. Go ahead and hit merge!